### ISSUE TITLE
TIMvision_2.0.8(AndroidTV).apk problem

### ISSUE THREAD

**Issue by lastway** [TheMiguelBi/APP_for_Amazon-Fire-Tv-Stick_BasicEdition#1]
Hi! 
I have installed the apk named in the title on a firestick tv, but it does not work for me. When I start it 4 little squares changing color appear on the screen and nothing else... I waited for several minutes, but nothing to do.

Where I am wrong? It seems there is a problem not only for me but for some other users. 
Thank you in advance.

**Comment by TheMiguelBi** [comment]
Yes, who uses it is aware of it. Currently there are no working remedies. 
As soon as I have news, I will be happy to update you!

**Comment by jecaho** [comment]
hi and thank you for your work
there isn't any update of the app that you can extract from cubovision/tim box?

**Comment by TheMiguelBi** [comment]
I do not own the cubovision Tim .... That could be the right way to try.

**Comment by alelale** [comment]
Hi 'TheMiguelBi'.
Today I've extracted from my 'TIM BOX' some apk files..among wich TimVision, Youtube, etc.
If these files can be useful for your work, how and where can I send them to you?
